publications/Multi-stage/flops/output.csv

```diff
@@ -0,0 +1,113 @@
+1000,256,inf,inf,inf,True,7115263,0,0,7115263
+1000,256,0,inf,inf,True,0,466454146,0,466454146
+1000,256,50,inf,inf,True,0,468935789,0,468935789
+1000,256,50,300,inf,True,0,32099570,7033071,39132641
+1000,256,50,100,inf,True,0,6090523,6987922,13078445
+1000,256,50,300,600,True,0,32382572,2570082,34952654
+1000,256,50,100,600,True,0,5888431,2770283,8658714
+1000,256,50,100,300,True,0,6190121,686317,6876438
+1500,256,inf,inf,inf,True,15761389,0,0,15761389
+1500,256,0,inf,inf,True,0,993931701,0,993931701
+1500,256,50,inf,inf,True,0,992940213,0,992940213
+1500,256,50,300,inf,True,0,31664422,14949260,46613682
+1500,256,50,100,inf,True,0,5764985,14983112,20748097
+1500,256,50,300,600,True,0,31843300,2570050,34413350
+1500,256,50,100,600,True,0,5562867,2563369,8126236
+1500,256,50,100,300,True,0,5666758,703603,6370361
+2000,256,inf,inf,inf,True,28077848,0,0,28077848
+2000,256,0,inf,inf,True,0,1570149841,0,1570149841
+2000,256,50,inf,inf,True,0,1577399492,0,1577399492
+2000,256,50,300,inf,True,0,32461476,28286358,60747834
+2000,256,50,100,inf,True,0,6037893,28354931,34392824
+2000,256,50,300,600,True,0,32086616,2613546,34700162
+2000,256,50,100,600,True,0,5867674,2593642,8461316
+2000,256,50,100,300,True,0,5948024,725891,6673915
+2500,256,inf,inf,inf,True,44109458,0,0,44109458
+2500,256,0,inf,inf,True,0,2721511975,0,2721511975
+2500,256,50,inf,inf,True,0,2695438065,0,2695438065
+2500,256,50,300,inf,True,0,32370697,44434582,76805279
+2500,256,50,100,inf,True,0,5948127,43890703,49838830
+2500,256,50,300,600,True,0,32253869,2483424,34737293
+2500,256,50,100,600,True,0,5928142,2610090,8538232
+2500,256,50,100,300,True,0,5955355,729978,6685333
+3000,256,inf,inf,inf,True,64113796,0,0,64113796
+3000,256,0,inf,inf,True,0,5932875080,0,5932875080
+3000,256,50,inf,inf,True,0,5761882163,0,5761882163
+3000,256,50,300,inf,True,0,32437242,64101349,96538591
+3000,256,50,100,inf,True,0,5947414,64572213,70519627
+3000,256,50,300,600,True,0,32116860,2488154,34605014
+3000,256,50,100,600,True,0,5924070,2482485,8406555
+3000,256,50,100,300,True,0,5949915,715312,6665227
+3500,256,inf,inf,inf,True,87241251,0,0,87241251
+3500,256,0,inf,inf,True,0,5890645659,0,5890645659
+3500,256,50,inf,inf,True,0,5877273979,0,5877273979
+3500,256,50,300,inf,True,0,31769604,87116757,118886361
+3500,256,50,100,inf,True,0,5873369,87709481,93582850
+3500,256,50,300,600,True,0,31623437,2476048,34099485
+3500,256,50,100,600,True,0,5953339,2487540,8440879
+3500,256,50,100,300,True,0,5928950,734191,6663141
+4000,256,inf,inf,inf,True,115018444,0,0,115018444
+4000,256,0,inf,inf,True,0,7463549730,0,7463549730
+4000,256,50,inf,inf,True,0,7454238194,0,7454238194
+4000,256,50,300,inf,True,0,31904357,114656303,146560660
+4000,256,50,100,inf,True,0,6141785,114794802,120936587
+4000,256,50,300,600,True,0,32491185,2491468,34982653
+4000,256,50,100,600,True,0,6159769,2625464,8785233
+4000,256,50,100,300,True,0,5923347,736517,6659864
+1000,256,inf,inf,inf,False,6618424,0,0,6618424
```

Comment:
This value seems to be different from the first row when Autotune=True. Not sure how much of a difference the random inputs make in this total FLOPS given that it's always float.

Can we try add a seed and run it multiple times to see if it's exactly the same?

Reply:
Please take a look at the new numbers